### PR TITLE
Add xml-test-run command documentation

### PR DESCRIPTION
**Pull request by elyezer**
Add a chapter to the documentation with more information about how to run the
xml-test-run command.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -395,6 +395,42 @@ Example using JSON format:
     not validate them. If an unexpected value is found, the command will fail
     with a stack trace showing the error.
 
+xml-test-run command
+++++++++++++++++++++
+
+The xml-test-run command generates an XML file suited to be imported by the
+Test Run XML importer. It takes a valid xUnit XML file and a Python test suite
+where test case IDs can be found and generates a resulting XML file with all
+the information the Test Run XML importer needs.
```
Comment (sthirugn) [inline docs/index.rst]: Please break this section in to bullets.  Something like:

```
It takes the following inputs:
* a Valid xUnit XML file
* a Python test suite where test case IDs can be found

It generates a resulting XML file with all the information necessary for the Test Run XML importer.
```

File: docs/index.rst [hunk 2]
```diff
@@ -395,6 +395,42 @@ Example using JSON format:
...
+
+The xml-test-run command only requires you to pass the path to the xUnit XML
+file, the path to the Python test suite source code, the Polarion user ID, the
+Polarion project ID and the output XML file path (it will override if the file
+already exists).
```
Comment (sthirugn) [inline docs/index.rst]: Please break this section in to different bullet items

File: docs/index.rst [hunk 3]
```diff
@@ -395,6 +395,42 @@ Example using JSON format:
...
+
+Even though ``--response-property`` is optional, it is highly recommended to
+pass it because will be easier to monitor the importer messages (which is not
+handled by Betelgeuse).
```
Comment (sthirugn) [inline docs/index.rst]: Add a note section for this paragraph

File: docs/index.rst [hunk 4]
```diff
@@ -395,6 +395,42 @@ Example using JSON format:
...
+        PROJECT \
+        output.xml
+
+The xml-test-run command can set test run custom fields.  The
+``--custom-fields`` option can be used with a ``key=value`` format or a JSON
+format as explained in `test-run command`_ section. Make sure to pass the the
+custom field ID (same as in Polarion) and its value. Make sure to pass custom
+field values as string since they will be converted to XML where there is no
+type information.
+
```
Comment (sthirugn) [inline docs/index.rst:445]: Move these two statements starting with `Make sure to pass` to a `warning` section

**Review by sthirugn — CHANGES_REQUESTED** [review]

**Comment by elyezer** [comment]
Comments addressed

**Comment by elyezer** [comment]
Since I am touching the documentation I decided to add a second commit to update the default token prefix from `@` to `:` on docs.

**Comment by sthirugn** [comment]
ACK

**Review by sthirugn — APPROVED** [review]

**Pull request merged**